.github/workflows/fossa-scanner.yaml

```
@@ -0,0 +1,15 @@
+name: "FOSSA Scanner"
+on:
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: fossas/fossa-action@main
+        with:
+          api-key: ${{secrets.FOSSAAPIKEY}}
+          team-name: "Public-Repos"
```

Changes:

- key should be changed as `team`
- Suggestion: Shall we configure the team name as a secret? 

```suggestion
          team: ${{secrets.FOSSATEAM}}
```